resources/views/news/index.blade.php

```diff
@@ -15,7 +15,7 @@
                         {{ $singleNews->author }}
 
                         <a href="{{ route('news.edit', $singleNews) }}">Edit</a>
-                        <form action="{{ route('news.destroy', $singleNews) }}" method="post">
+                        <form action="{{ route('news.destroy', $singleNews) }}">
```

Comment:
I would still prefer keeping the method="post" in cases where request isn't a get request, but if that's working, then just remember to keep it consistent across all forms.

Reply:
Maybe create issue for delete method="post" ?